stacks-signer/src/signerdb.rs

```diff
@@ -826,6 +826,12 @@ impl SignerDb {
         try_deserialize(result)
     }
 
+    /// Return the count of signed blocks in a tenure (identified by its consensus hash)
+    pub fn get_signed_block_count_in_tenure(&self, tenure: &ConsensusHash) -> Result<i64, DBError> {
+        let query = "SELECT COUNT(*) FROM blocks WHERE consensus_hash = ? AND signed_over = 1";
+        query_count(&self.db, query, [tenure])
+    }
```

Comment:
I think this operates on locally accepted blocks (i.e., this signer signed off on the proposal), but I think it should instead operate on _globally accepted_ blocks (so it would be okay to reorg the tenure if there were 1 globally accepted and 1 locally accepted proposals, but not if there were 2 globally accepted proposals).

Reply:
It definitely makes sense, thanks